std/os/freebsd/index.zig

```diff
@@ -699,6 +718,14 @@ pub fn nanosleep(req: *const timespec, rem: ?*timespec) usize {
     return errnoWrap(c.nanosleep(req, rem));
 }
 
+pub fn clock_gettime(clk_id: i32, tp: *timespec) usize {
+    return errnoWrap(c.clock_gettime(clk_id, tp));
+}
+
+pub fn clock_getres(clk_id: i32, tp: *timespec) usize {
+    return clock_gettime(clk_id, tp);
```

Comment:
shouldn't this be a call to `c.clock_getres`?

Reply:
ops, you're right.
Thank you!